cumulusci/core/keychain/BaseProjectKeychain.py

```diff
@@ -11,6 +12,15 @@
 from cumulusci.core.sfdx import sfdx
 
 
+DEFAULT_CONNECTED_APP = ConnectedAppOAuthConfig(
+    {
+        "client_id": "3MVG9i1HRpGLXp.or6OVlWVWyn8DXi9xueKNM4npq_AWh.yqswojK9sE5WY7f.biP0w7bNJIENfXc7JMDZGO1",
```

Comment:
Hardcoding this is ok, as we don't expect to have to ever switch from the default connected app, correct?

Reply:
That's the idea. If we did need to delete the existing connected app, users would need to upgrade to a new version of CumulusCI or configure their own connected app (the way they've needed to until now).